src/views/MoviesPage.vue

```diff
@@ -0,0 +1,130 @@
+<script>
+import { defineComponent } from 'vue';
+
+import BaseTextInput from '@/components/global/BaseTextInput.vue';
+import BaseSelect from '@/components/global/BaseSelect.vue';
+import MoviesContainer from '@/components/MoviesPage/MoviesContainer.vue';
+
+import { useMoviesStore } from '@/store/movies';
+const moviesStore = useMoviesStore();
+
+export default defineComponent({
+  name: 'MoviesPage',
+  components: {
+    BaseTextInput,
+    BaseSelect,
+    MoviesContainer
+  },
+  data() {
+    return {
+      searchQuery: '',
+      movieCategory: ''
+    };
+  },
+  computed: {
+    movies() {
+      return moviesStore.allMovies;
+    },
+    genres() {
+      return [...new Set(moviesStore.genres)];
```

Comment:
💪  
You can consider to do it in the store if you don't need redundant genres in any case 👌

Reply:
And that's done to!